metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures Nagios NRPE client'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '1.4.13'
+version           '1.4.14'
```

Comment:
This should become the 1.5 release probably and a note should be added to the CHANGELOG that this will be a breaking change if anyone is using the existing array in a role or environment.